src/pages/home/HeroSection.jsx

```
@@ -9,15 +9,15 @@ const HeroSection = () => {
+        <h2 className="text-4xl font-semibold text-white font-oswald">
+          STYLE. TECH. BEAUTY. HOME. ALL IN ONE PLACE
         </h2>
         <p className="text-base text-white font-open-sans">
-          Explore a wide range of high-quality products handpicked just for you
-          &mdash; all categories, all savings, delivered to your door.
+          EXPLORE A WIDE RANGE OF HIGH-QUALITY PRODUCTS HANDPICKED JUST FOR YOU
+          &mdash; ALL CATEGORIES, ALL SAVINGS, DELIVERED TO YOUR DOOR.
         </p>
         <Button
-          children="Start Shopping"
+          children="START SHOPPING"
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)